modules/ROOT/pages/introduction.adoc

```diff
@@ -150,6 +150,10 @@ a| High_limit (xref:tools/neo4j-admin/neo4j-admin-store-info.adoc#neo4j-admin-st
 |
 | {check-mark}
 
+a| Block format (xref:tools/neo4j-admin/neo4j-admin-store-info.adoc#neo4j-admin-store-block[New format])
```

Comment:
We'll need to update the link if we move the formats from **Display store information** to **Store formats** page.